pos_financial_risk/static/src/xml/PaymentScreen.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+    <t t-inherit="point_of_sale.PaymentScreen" t-inherit-mode="extension" owl="1">
+        <xpath
+            expr="//t[@t-foreach='payment_methods_from_config']"
+            position="attributes"
+        >
+            <attribute name="t-foreach">paymentMethodsUnlock</attribute>
+        </xpath>
+        <xpath expr="//div[hasclass('paymentmethods')]" position="after">
+            <div class="paymentmethods" t-if="paymentMethodsLock.length > 0">
+                <p class="title-category">Blocked due to credit limit reached</p>
+                <t
+                    t-foreach="paymentMethodsLock"
+                    t-as="paymentMethod"
+                    t-key="paymentMethod.id"
+                >
+                    <div class="button paymentmethod">
+                        <div
+                            class="payment-name"
+                            style="background: rgba(255, 0, 0, 0.6); cursor: not-allowed;"
```

Comment:
It's not really a good practice to add inline styles.
The convention is to add styles through css or scss files instead